dev-libs/zziplib/patches/zziplib-0.13.67.patchset

```diff
@@ -30,3 +30,73 @@ index 69070df..410487a 100644
 -- 
 2.7.0
 
+
+From 0c3f92c5f5080eba05278dca3d25c3860ad315f0 Mon Sep 17 00:00:00 2001
+From: Leorize <alaviss@users.noreply.github.com>
+Date: Mon, 4 Dec 2017 13:34:19 +0700
+Subject: test: set LIBRARY_PATH instead of LD_LIBRARY_PATH
+
+
+diff --git a/test/Makefile.am b/test/Makefile.am
+index 250651d..d7b31b9 100644
+--- a/test/Makefile.am
++++ b/test/Makefile.am
+@@ -70,7 +70,7 @@ zzshowme$(EXEEXT) : zzipself$(EXEEXT) zzipsetstub$(EXEEXT)
+ 
+ check-sfx : zzshowme$(EXEEXT)
+ 	echo :$@: "./zzshowme readme >readme.out 2>readme.err" 
+-	export LD_LIBRARY_PATH="../zzip/.libs:$$LD_LIBRARY_PATH" \
++	export LIBRARY_PATH="../zzip/.libs:$$LIBRARY_PATH" \
+ 	; ./zzshowme readme >readme.out 2>readme.err ; true
+ 	@ echo 'diff readme.out $(README) || grep "libzzip-" readme.err' \
+ 	; if test -s readme.out ; then diff readme.out $(README) \
+-- 
+2.7.0
+
+
+From 29d8ccf8444ca965148f778285aea2367c336391 Mon Sep 17 00:00:00 2001
+From: Leorize <alaviss@users.noreply.github.com>
+Date: Mon, 4 Dec 2017 17:20:50 +0700
+Subject: test: use a loop instead of globbing
+
+10000 files on argv, won't work
+
+diff --git a/test/zziptests.py b/test/zziptests.py
+index 6e9df78..0f73300 100644
+--- a/test/zziptests.py
++++ b/test/zziptests.py
+@@ -199,7 +199,7 @@ class ZZipTest(unittest.TestCase):
+     filename = os.path.join(tmpdir,"README")
+     filetext = self.readme()
+     self.mkfile(filename, filetext)
+-    shell("{exe} -n README ../{zipfile} ??*.* README".format(**locals()), cwd=tmpdir)
++    shell("for f in ??*.*; do {exe} -un README ../{zipfile} $f README; done".format(**locals()), cwd=tmpdir)
```

Comment:
I think xargs could be used to make this less slow (it can pass the arguments by groups of arbitrary size to another process).